### DISCUSSION TITLE
How to prevent file name changes when uploading images with Curator?

### DISCUSSION THREAD

**Discussion by letho04** [awcodes/filament-curator#557] [Q&A]
Hello, I am looking for a setting in Curator that keeps the original filename when uploading images instead of automatically generating a new filename by default. 
I found a way to configure this for each individual resource using the function . 
However, this is quite inconvenient when I want to upload multiple files by clicking the Media button in the sidebar (the Curator plugin registered in the panel). 
So, I would like to ask if there is any function that supports this?->preserveFilenames()

**Comment by awcodes** [comment]
There's a setting in the config 'should_preserve_filenames'

**Reply by letho04** [reply]
Thank you, I have found and configured it as desired.
